### PR TITLE
[zh-cn] Update CSS layout cookbook mainpage

### PR DESCRIPTION
**Pull request by jasonren0403**

### CODE REVIEW

File: files/zh-cn/web/css/layout_cookbook/index.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+| [卡片](/zh-CN/docs/Web/CSS/Layout_cookbook/Card)                                     | 卡片组件，显示在卡片网格中                                                                  | [CSS 网格](/zh-CN/docs/Web/CSS/CSS_Grid_Layout)                                                                                                   |
+| [网格包装器](/zh-CN/docs/Web/CSS/Layout_cookbook/Grid_wrapper)                     | 用于在一个中央包装器内对齐网格内容，同时也允许项目脱离。 | [CSS 网格](/zh-CN/docs/Web/CSS/CSS_Grid_Layout)                                                                                                      |
 
-## Contribute a Recipe
+## 贡献布局专题
 
-As with all of MDN we would love you to contribute a recipe in the same format as the ones shown above. [See this page](/zh-CN/docs/user:chrisdavidmills/Layout_Cookbook_Home/Contribute_a_recipe) for a template and guidelines for writing your own example.
+作为 MDN 的一员，我们希望你能贡献一份与上图所示相同格式的布局小专题。[参见本页](/zh-CN/docs/Web/CSS/Layout_cookbook/Contribute_a_recipe)以获取撰写自己的模板和指引的方法。
```
Comment (jasonren0403) [inline files/zh-cn/web/css/layout_cookbook/index.md:29]: 第一句 `As with all of MDN we would love you to contribute a recipe in the same format as the ones shown above.` 并不确定是否该这样翻

File: files/zh-cn/web/css/layout_cookbook/index.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
 ---
-title: CSS Layout cookbook
+title: CSS 布局手册
```
Comment (jasonren0403) [inline files/zh-cn/web/css/layout_cookbook/index.md:2]: 本文的 `cookbook` 都处理成了“手册”、`recipe` 都处理为了“专题”

File: files/zh-cn/web/css/layout_cookbook/index.md [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
 slug: Web/CSS/Layout_cookbook
 ---
 
 {{CSSRef}}
 
-CSS layout cookbook 这本书旨在汇集常见的布局样式，这些布局你可能在自己的网站中会用到。这些布局样式不仅仅可以可以用来开始一个项目，而且提供了具体的使用方法，作为开发人员，你可以做出合适的选择。
+CSS 布局手册旨在汇集常见的布局样式，这些布局你可能在自己的网站中会用到。这些布局样式不仅仅可以用来开始一个项目，而且提供了具体的使用方法，作为开发人员，你可以做出合适的选择。
 
-> **备注：** 如果你是首次接触 CSS 布局，你可以先看一下[CSS layout learning module](/zh-CN/docs/Learn/CSS/CSS_layout)，书中的基础知识有利于你更好的使用本书中的布局样式。
+> **备注：** 如果你是首次接触 CSS 布局，你可以先看一下[CSS 布局学习模块](/zh-CN/docs/Learn/CSS/CSS_layout)，其中的基础知识有利于你更好的使用本书中的布局样式。
```
[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/web/css/layout_cookbook/index.md:10]:
```suggestion
> **备注：** 如果你是首次接触 CSS 布局，你可以先看一下 [CSS 布局学习模块](/zh-CN/docs/Learn/CSS/CSS_layout)，其中的基础知识有利于你更好的使用本书中的布局样式。
```

File: files/zh-cn/web/css/layout_cookbook/index.md (hunk 1 above)
Comment (jasonren0403) [reply files/zh-cn/web/css/layout_cookbook/index.md:29]:
```suggestion
作为 MDN 的一员，我们希望你能贡献一份与上图所示相同格式的布局小专题。[参见此页](/zh-CN/docs/Web/CSS/Layout_cookbook/Contribute_a_recipe)以获得模板和编写自己的示例的指南。
```

Comment (yin1999) [reply files/zh-cn/web/css/layout_cookbook/index.md:29]: 这句话里面的“编写示例”应该同时修饰了模板和指南这两者：

```suggestion
作为 MDN 的一员，我们希望你能贡献一份与上图所示相同格式的布局专题。[参见此页](/zh-CN/docs/Web/CSS/Layout_cookbook/Contribute_a_recipe)以获得编写示例的模板和指南。
```

Comment (jasonren0403) [reply files/zh-cn/web/css/layout_cookbook/index.md:29]:
```suggestion
作为 MDN 的一员，我们希望你能贡献一份与上表所示相同格式的布局专题。[参见本页](/zh-CN/docs/Web/CSS/Layout_cookbook/Contribute_a_recipe)以获得编写示例的模板和指南。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**